dags/pubmed_migration.py

```diff
@@ -17,7 +17,8 @@
 # Setting up MongoDB hook to mlab server
 mongodb_hook = MongoHook('mongo_default')
 ftp_conn_id = "pubmed_ftp"
-
+s3file = 'test'
```

Comment:
This is a little pedantic, but this wouldn't be a file right. it would be a directory or path prefix?. Also, we shouldn't commit "test" code.